plugins/package-managers/pub/src/funTest/kotlin/PubFunTest.kt

```diff
@@ -70,7 +68,7 @@ class PubFunTest : WordSpec({
             val definitionFile = getAssetFile("projects/synthetic/multi-module/pubspec.yaml")
             val expectedResultFile = getAssetFile("projects/synthetic/pub-expected-output-multi-module.yml")
 
-            val analyzerResult = analyze(definitionFile.parentFile).patchPackages()
+            val analyzerResult = analyze(definitionFile.parentFile).analyzer!!.result.patchPackages()
```

Comment:
Previously this code path called `.withResolvedScopes()`, now it doesn't anymore.